<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceSettingsPane</class>

 <widget class="QWidget" name="InterfaceSettingsPane">
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>620</height>
   </size>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed"/>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>

  <layout class="QVBoxLayout" name="layoutRoot">
   <property name="spacing"><number>10</number></property>
   <property name="leftMargin"><number>10</number></property>
   <property name="topMargin"><number>10</number></property>
   <property name="rightMargin"><number>10</number></property>
   <property name="bottomMargin"><number>10</number></property>

   <item>
    <layout class="QHBoxLayout" name="layoutMain">
     <property name="spacing"><number>30</number></property>

     <!-- ================= LEFT PANEL ================= -->
     <item>
      <widget class="QWidget" name="leftPane">
       <property name="minimumWidth"><number>480</number></property>
       <property name="maximumWidth"><number>480</number></property>

       <layout class="QVBoxLayout" name="layoutLeft">
        <property name="spacing"><number>10</number></property>

        <!-- Appearance -->
        <item>
         <widget class="QGroupBox" name="groupAppearance">
          <property name="title"><string>Appearance</string></property>
          <layout class="QGridLayout" name="gridAppearance">
           <item row="0" column="0"><widget class="QLabel" name="lblThemeMode"><property name="text"><string>Theme mode</string></property></widget></item>
           <item row="0" column="1"><widget class="QComboBox" name="comboThemeMode"/></item>
           <item row="1" column="0"><widget class="QLabel" name="lblCustomQss"><property name="text"><string>Custom theme (QSS)</string></property></widget></item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="layoutCustomQss">
             <item><widget class="QLineEdit" name="editCustomQssPath"/></item>
             <item><widget class="QPushButton" name="btnLoadCustomQss"><property name="text"><string>Browseâ€¦</string></property></widget></item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>

        <!-- Language -->
        <item>
         <widget class="QGroupBox" name="groupLanguage">
          <property name="title">
           <string>Language</string>
          </property>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="lblLanguage">
             <property name="text">
              <string>Interface language</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboLanguage">
             <property name="minimumWidth">
              <number>160</number>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>


        <!-- Fonts -->
        <item>
         <widget class="QGroupBox" name="groupFonts">
          <property name="title"><string>Fonts</string></property>
          <layout class="QGridLayout">
           <item row="0" column="0"><widget class="QLabel" name="lblAppFont"><property name="text"><string>Application font</string></property></widget></item>
           <item row="0" column="1"><widget class="QFontComboBox" name="fontComboApp"/></item>
           <item row="1" column="0"><widget class="QLabel" name="lblAppFontSize"><property name="text"><string>Interface font size</string></property></widget></item>
           <item row="1" column="1"><widget class="QSpinBox" name="spinAppFontSize"/></item>
          </layout>
         </widget>
        </item>

        <!-- Toolbar -->
        <item>
         <widget class="QGroupBox" name="groupToolbar">
          <property name="title"><string>Toolbar</string></property>
          <layout class="QGridLayout">
           <item row="0" column="0"><widget class="QRadioButton" name="radioToolbarIcons"><property name="text"><string>Icons only</string></property></widget></item>
           <item row="1" column="0"><widget class="QRadioButton" name="radioToolbarText"><property name="text"><string>Text only</string></property></widget></item>
           <item row="2" column="0"><widget class="QRadioButton" name="radioToolbarBoth"><property name="text"><string>Icons and text</string></property></widget></item>
           <item row="3" column="0">
            <layout class="QHBoxLayout">
             <item><widget class="QLabel" name="lblIconSize"><property name="text"><string>Icon size</string></property></widget></item>
             <item><widget class="QSpinBox" name="spinToolbarIconSize"/></item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>

        <!-- Thumbnails -->
        <item>
         <widget class="QGroupBox" name="groupThumbnails">
          <property name="title"><string>Page thumbnails</string></property>
          <layout class="QHBoxLayout">
           <item><widget class="QLabel" name="lblThumbSize"><property name="text"><string>Thumbnail size (px)</string></property></widget></item>
           <item>
           <widget class="QSpinBox" name="spinThumbSize">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>130</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>

           </item>
          </layout>
         </widget>
        </item>



        <!-- ================================================ -->
        <!-- Restore defaults button                          -->
        <!-- ================================================ -->
        <item>
         <layout class="QHBoxLayout" name="layoutReset">
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </spacer>
          </item>

          <item>
           <widget class="QPushButton" name="btnRestoreDefaults">
            <property name="text">
             <string>Restore defaults</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>

       </layout>
      </widget>
     </item>

     <!-- ================= RIGHT PANEL ================= -->
     <item>
      <widget class="QWidget" name="previewPane">
       <property name="minimumWidth"><number>1060</number></property>
       <layout class="QVBoxLayout" name="layoutPreviewContainer">

        <item>
         <widget class="QFrame" name="previewRoot">
          <property name="frameShape"><enum>QFrame::StyledPanel</enum></property>
          <layout class="QVBoxLayout">

            <item>
             <widget class="QLabel" name="lblPreviewTitle">
              <property name="text">
               <string>OCRtoODT GUI</string>
              </property>

              <property name="alignment">
               <set>Qt::AlignHCenter</set>
              </property>

              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
              </property>

             </widget>
            </item>



           <!-- Toolbar -->
           <item>
            <layout class="QHBoxLayout" name="layoutPreviewToolbar">
             <item><widget class="QToolButton" name="btnPrevOpen"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevClear"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevSettings"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevRun"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevStop"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevExport"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevAbout"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><widget class="QToolButton" name="btnPrevHelp"><property name="autoRaise"><bool>true</bool></property></widget></item>
             <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
            </layout>
           </item>

           <!-- Central -->
           <item>
            <layout class="QHBoxLayout" name="layoutPreviewCentral">

             <!-- Pages -->
             <item>
              <widget class="QFrame" name="previewThumbnails">
               <property name="minimumWidth"><number>220</number></property>
               <property name="maximumWidth"><number>220</number></property>
               <layout class="QVBoxLayout">
                <item><widget class="QLabel" name="lblPagesTitle"><property name="text"><string>Pages</string></property></widget></item>
                <item><widget class="QListWidget" name="listPreviewPages"/></item>
               </layout>
              </widget>
             </item>

             <!-- Document -->
             <item>
              <widget class="QFrame" name="previewDocument">
               <layout class="QVBoxLayout" name="layoutPreviewDocument">

                <item>
                 <widget class="QLabel" name="lblDocumentPlaceholder">
                  <property name="alignment"><set>Qt::AlignCenter</set></property>
                  <property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Expanding"/></property>
                  <property name="minimumSize"><size><width>430</width><height>440</height></size></property>
                 </widget>
                </item>

                <item>
                 <layout class="QHBoxLayout" name="layoutPreviewZoom">
                  <item><widget class="QPushButton" name="btnZoomIn"><property name="text"><string>+</string></property><property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Fixed"/></property><property name="minimumHeight"><number>32</number></property></widget></item>
                  <item><widget class="QPushButton" name="btnZoomOut"><property name="text"><string>-</string></property><property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Fixed"/></property><property name="minimumHeight"><number>32</number></property></widget></item>
                  <item><widget class="QPushButton" name="btnZoomFit"><property name="text"><string>Fit</string></property><property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Fixed"/></property><property name="minimumHeight"><number>32</number></property></widget></item>
                  <item><widget class="QPushButton" name="btnZoom100"><property name="text"><string>100%</string></property><property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Fixed"/></property><property name="minimumHeight"><number>32</number></property></widget></item>
                 </layout>
                </item>

               </layout>
              </widget>
             </item>

             <!-- OCR Result -->
             <item>
              <widget class="QFrame" name="previewResult">
               <property name="minimumWidth"><number>260</number></property>
               <property name="maximumWidth"><number>360</number></property>
               <layout class="QVBoxLayout">
                <item><widget class="QLabel" name="lblResultTitle"><property name="text"><string>OCR Result</string></property></widget></item>
                <item><widget class="QPlainTextEdit" name="editPreviewResult"/></item>
               </layout>
              </widget>
             </item>

            </layout>
           </item>

          </layout>
         </widget>
        </item>

       </layout>
      </widget>
     </item>

    </layout>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>

